docs/UserGuide.md

```diff
@@ -109,7 +109,7 @@ Examples:
 * `mark A0123456G DONE`
 * `mark A0234567H YTS`
 
-### Searching keyword: `find`
```

Comment:
This should be included in the PR description or the commit message but no worries!